src/extension.spec.ts

```diff
@@ -94,15 +94,13 @@ describe('extension', () => {
 
     describe('RDB', () => {
         const extensionInstance = extension.extension as any;
-        let config;
+        const config = {} as any;
```

Comment:
Can we give this a better type? It's probably `BrightScriptLaunchConfiguration` right?
```suggestion
        const config = {} as BrightScriptLaunchConfiguration;
```

Reply:
updated